test_generators/epoch_processing/main.py

```diff
@@ -0,0 +1,38 @@
+from typing import Callable, Iterable
+
+from eth2spec.test.epoch_processing import (
+    test_process_crosslinks,
+    test_process_registry_updates
+)
+
+from gen_base import gen_runner, gen_suite, gen_typing
+from gen_from_tests.gen import generate_from_tests
+from preset_loader import loader
+from eth2spec.phase0 import spec
+
+
+def create_suite(transition_name: str, config_name: str, get_cases: Callable[[], Iterable[gen_typing.TestCase]]) \
+        -> Callable[[str], gen_typing.TestSuiteOutput]:
```

Comment:
nitpick: IMO avoid `\` here
```python
def create_suite(
        transition_name: str,
        config_name: str,
        get_cases: Callable[[],
        Iterable[gen_typing.TestCase]]) -> Callable[[str], gen_typing.TestSuiteOutput]:
```

Reply:
My IDE gives a PEP8 warning with that suggestion.

Reply:
And formatter adds 20 spaces of indentation to the last line of that suggestion, and then it's happy... I'd rather keep it as-is.